app/ui-react/syndesis/src/modules/connections/components/Connections.tsx

```diff
@@ -13,11 +13,17 @@ import { Translation } from 'react-i18next';
 import { UIContext } from '../../../app';
 import { ApiError, EntityIcon } from '../../../shared';
 
+export interface IConnectionOverview extends ConnectionOverview {
```

Comment:
Yes something like this, only this should be in the `extra.d.ts` and should be referenced by the `useConnection`, etc. components/hooks to explain their return type.

Oh and no need to specify `board`, it's already part of the `ConnectionOverview` interface.

Reply:
I pushed up some changes, it's a bit difficult to test with staging being down. I moved the interface and the `board` prop, updated the reference in `useConnection` and `Connections.tsx`, but if I missed anything let me know.